apps/ledger-live-desktop/src/renderer/styles/StyleProviderV3.tsx

```diff
@@ -2,12 +2,12 @@ import "@ledgerhq/react-ui/assets/fonts";
 import React, { useMemo } from "react";
```

Comment:
in this file I did 2 things:

- renaming of slightly confusing names
- cleanup of useless logic (the `const v3SelectedPalettes = ...` and `palettesAny: any = palettes`)
- adding `v2Palettes` to `colors` because in one case (`ContrastChoice` component) we need to access a v2 background color in a v3 component which therefore cannot access that color (because style provider context etc.)